src/wp-admin/install.php

```diff
@@ -408,6 +411,10 @@ function display_setup_form( $error = null ) {
 			// TODO: Poka-yoke.
 			display_setup_form( __( 'Sorry, that is not a valid email address. Email addresses look like <code>username@example.com</code>.' ) );
 			$error = true;
+		}elseif ( ! isset( $_POST['installation_field'] ) || ! wp_verify_nonce( $_POST['installation_field'], 'wordpress_setup' ) ) {
```

Comment:
```suggestion
		} elseif ( ! isset( $_POST['installation_field'] ) || ! wp_verify_nonce( $_POST['installation_field'], 'wordpress_setup' ) ) {
```
This will fix the Coding Standards failure and allow CI to pass 🙂